tools/dreport.d/openpower.d/plugins.d/vpd_data

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+#
+# config: 23 25
+# @brief: Collect VPD persistent data
+#
+
+. $DREPORT_INCLUDE/functions
+
```

Comment:
There is already "bad vpd" placed in IBM layer , Please check with VPD owner and keep both plugin in same layer.

Also  follow the plugin format
desc =""
dir_name = ""